node/npm/glide/index.ts

```diff
@@ -106,7 +106,6 @@ function initialize() {
         ListDirection,
         ExpireOptions,
         FlushMode,
-        GeoUnit,
```

Comment:
Why this change?

Reply:
A merge artifact - it got duplicated (see line 90). This code does not compile when we run tests, so no one sees the errors.